### PR TITLE
add get model in nnframe doc

### PR DESCRIPTION
**Pull request by dding3**

### CODE REVIEW

File: docs/readthedocs/source/doc/DLlib/Overview/nnframes.md [hunk 1]
```diff
@@ -203,6 +203,17 @@ nn_model = NNModel(bigDLModel)
 
 Meanwhile, for advanced use cases (e.g. model with multiple input tensor), `NNModel` supports: `setSamplePreprocessing(value: Preprocessing[Any, Sample[T]])`to directly compose Sample according to user-specified Preprocessing.
 
+We can get model from `NNModel` by:
+**Scala:**
```
Comment (hkvision) [inline docs/readthedocs/source/doc/DLlib/Overview/nnframes.md:207]: new line before scala

**Review by hkvision — APPROVED** [review]

**Comment by dding3** [comment]
jekins pass. http://10.112.231.51:18888/view/ZOO-PR/job/BigDL-PR-Dllib-Validation/161/

File: docs/readthedocs/source/doc/DLlib/Overview/nnframes.md [hunk 2]
```diff
@@ -203,6 +203,18 @@ nn_model = NNModel(bigDLModel)
...
+We can get model from `NNModel` by: 
+
+**Scala:**
+```scala
+val model = nnModel.getModel()
+```
+
+**Python:**
+```python
+model = nn_model.model
```
Comment (jason-dai) [inline docs/readthedocs/source/doc/DLlib/Overview/nnframes.md:215]: shall we add `get_model` to python?

Comment (dding3) [reply docs/readthedocs/source/doc/DLlib/Overview/nnframes.md:215]: ok

**Pull request merged**